src/core/thread/mle_router.cpp

```diff
@@ -2973,7 +2989,8 @@ otError MleRouter::SendChildUpdateResponse(Child *aChild, const Ip6::MessageInfo
 
     SuccessOrExit(error = SendMessage(*message, aMessageInfo.GetPeerAddr()));
 
-    otLogInfoMle(GetInstance(), "Sent Child Update Response to child");
+
+    LogMleMessage("Send Child Update Response to child", aMessageInfo.GetPeerAddr());
```

Comment:
Suggestion: How about adding the child rloc as well?

Reply:
Child may or may not have a valid RLOC.  I can handle both cases.